CHANGELOG.next.asciidoc

```diff
@@ -25,7 +25,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 *Heartbeat*
 
 *Metricbeat*
-
+- Allow filtering on AWS tags by more than 1 value per key
```

Comment:
please, add a reference to the PR in the format `{pull}32775[32775]`

also the changelog entry should go under [Added -> Metricbeat](https://github.com/elastic/beats/blame/9b3343a8aacd3ad6f9c16b63f7d0971f05f1506c/CHANGELOG.next.asciidoc#L134) and not under "Breaking changes"

Reply:
fixed